src/elements/toniq-dropdown/toniq-dropdown.story.tsx

```diff
@@ -0,0 +1,44 @@
+import {ComponentMeta} from '@storybook/react';
+import {TypedEvent} from 'element-vir';
+import React from 'react';
+import {ToniqDropdown} from '../react-components';
+
+const componentStoryMeta: ComponentMeta<typeof ToniqDropdown> = {
+    title: 'Elements/Toniq Dropdown',
+    component: ToniqDropdown,
+};
+
+const list = [
+    {
+        value: 1,
+        label: 'Option 1',
+    },
+    {
+        value: 2,
+        label: 'Option 2',
+    },
+    {
+        value: '3',
+        label: 'Option 3',
+    },
+];
+
+const handleChange = (event: TypedEvent) => {
+    console.log(event.detail);
```

Comment:
Hook up the event to actions like the button story does now:
![Screen Shot 2022-06-03 at 09 17 03](https://user-images.githubusercontent.com/1205860/171882964-122c36e5-64dd-4ab5-9231-3746f470ad2a.png)

Reply:
Hook it up all all the examples here and remove the `onSelectChange Event` example